tests/sentry/api/endpoints/test_group_attachments.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import absolute_import
+
+import six
+from six.moves.urllib.parse import urlencode
+
+from sentry.models import EventAttachment, File
+from sentry.testutils import APITestCase
+
+
+class GroupEventAttachmentsTest(APITestCase):
+    def create_attachment(self, type=None):
```

Comment:
This looks very similar to `sentry.testutils.factories:Factories.create_event_attachment` Could we leverage that in this test case too?

Reply:
Good point, we actually need to fix `create_event_attachment` in factories, since this now creates an attachment without type, etc. Will fix this in a follow-up. Thanks for pointing it out!